tests/smoke/base.sh

```diff
@@ -40,21 +40,21 @@ export PATH=$(echo $PATH| sed -E "s/[^:]+\/micromamba\/[^:]+//g")
 export PATH=$(echo $PATH| sed -E "s/[^:]+\/anaconda3\/[^:]+//g")
 export PATH="${CONDA_PREFIX}:${CONDA_PREFIX}/bin:$PATH"
 echo "[II] included env conda to the PATH"
-COMMAND_PREFIX= 
+COMMAND_PREFIX=
 if command -v poetry &> /dev/null; then
   poetry install
 elif command -v flit &> /dev/null; then
   flit install
 elif command -v meson &> /dev/null; then
-  pip install -e .
+  pip install .
 elif command -v pdm &> /dev/null; then
   pdm install
 elif command -v hatch &> /dev/null; then
   COMMAND_PREFIX="hatch run"
 elif command -v maturin &> /dev/null; then
-  pip install -e .
+  pip install .
 elif [ "$(pip list|grep -c scikit_build_core)" -ne "0" ]; then
-  pip install -e .
+  pip install .
 else
     # use setuptools
   pip install --editable .
```

Comment:
Should `setuptools` also not have an editable install?

Reply:
you're right! good catch ... I just search by `-e` in the code .. I forgot to search bybout `--editable` XD